### ISSUE TITLE
ACE 3 - Rig Protection is inconsitent

### ISSUE THREAD

**Issue by AdlofHilder** [acemod/ACE3#7539]
Conditions of testing -
All Rigs tested were vanilla ARMA rigs. I tested a few CUP rigs and they seemed to have the same behaviour.  

All shots were directly into the centre of the chest

The CUP AS Val was used as the test weapon. (9x39)

As ACE simulates lots of medical states, the easiest things to test were KO time and injury generation (these are the most consistent from what I can tell)

I would allow them to bandage their wounds if two limbs started bleeding so they would not bleed out during testing.  

Results - 

Most level 3 rigs seemed to be a three-shot KO on average with AS Val Point blank -- This seems a little too tough IMO

From what I can tell the majority of Level 3 and some level 4 allows AS Val rounds to hit with medium wounds.  This isn't really reasonable considering it is point-blank, and they usually wake up.  

Please note: There are rigs with higher ballistic protection that I have tested, for instance, the "Carrier Rig" took around 6 shots, but generated small and medium velocity wounds which KO'd the guy faster than some rigs with "lesser" protection. I.E(The Carrier GL Rig)

**The Carrier GL Rig**

I failed to KO the 2 test subjects within a whole mag, each. One wearing the new contact woodland version, and one wearing the base game black rig, 

The Carrier GL Rigtook 30 rounds from the AS Val Point blank. Here are the injuries:
Left Arm - 12 Minor bruises and 2 Partial minor avulsions
Torso - 8 Minor bruises 
Right Arm - 4 Minor avulsions

I couldn't believe this so I tried again, this time with 20 Rounds.
Left Arm - 6 Minor bruises and 1 Partial minor avulsions
Torso - 5 Minor bruises and 1 Partial minor avulsions

I then tried the 7.62 VS-121 for 10 rounds 
Left Arm - 2 Minor bruises and 2 Partial minor avulsions
Torso - 2 Minor bruises 
Right Arm - 3 Minor bruises and 1 Partial minor avulsions

Summary
I am happy to help test **every** rig in the game, all I know is;
This is obviously a symptom of a system producing unintended results 
This isn't the only rig affected.
I believe it is contributing to this effect too (https://youtu.be/-najIt8H2OY?t=179)

Footnote:
I've likely forgotten to include some details. but I should be very responsive to any questions!
I LOVE ACE, and just want to help make it perfect for all of us!
Can't thank you guys enough for getting such an in-depth system this far, I'm just trying to make sure this doesn't go to waste! 
I'm happy to reproduce results with video capture if desired!

**Comment by commy2** [comment]
The level X you read in the UI is a bullshit tooltip that has no connection with the actual `armor` and `passThrough` values in protection config.

**Comment by commy2** [comment]
You commented [here](https://github.com/acemod/ACE3/issues/7349#issuecomment-589903894). How much overlap is there with both issue tickets? I don't like the tracker flooded with the same thing over and over again.

I see no reason to further test vests until https://github.com/acemod/ACE3/pull/7415 is merged.

**Comment by AdlofHilder** [comment]
I had no idea the UI didn't correlate, I'll take a look at the values in the protection config later to see if there's any correlation between the ones that soak up rounds, 

Your concern is vaild, however that issue is about the Sum of Trauma, just one of the comments blamed the weapons - which I found wasn't the case. The OP may have actually come across this without realising which made his issue worse, but the core complaints are different.

Hope this helps!

**Comment by AdlofHilder** [comment]
@commy2 Where can I find the actual armour and passThrough values in protection config?
I've been looking further into this and helmets perform vastly different from each other, not to go down the rabbit hole here... but I would love to see ACE indicate the level of protection that it would provide - seeing that it is built over the ACE medical system.

I'm happy to provide my findings on helmets I've tested thus far, however, I don't know whether  this is of any interest to you or the team. 

TL;DR - As the UI cannot provide info on what protection I get from helmets, is this something that can be provided from ACE? 

Sorry to be a bother! :)

**Comment by commy2** [comment]
>Where can I find the actual armour and passThrough values in protection config?

Ingame config viewer.

**Comment by severgun** [comment]
> TL;DR - As the UI cannot provide info on what protection I get from helmets/rigs, is this something that can be provided from ACE?

ACE Arsenal

**Comment by AdlofHilder** [comment]
@severgun 
> Visually - ACE Arsenal. Values - config viewer.
I appreciate the response, however, isn't it the case that those visual indicators in the UI are wrong, as commy said? 
I have since looked over the values but haven't took to investigate further since the issue with armour in 7415 may change my results entirely.

**Comment by severgun** [comment]
1) You can download 3.13.1-RC and test it with 7415 merged.
2) @commy2 said about level/class protection tooltips in items description.
As much as I know bars in arsenal use values from configs
